python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -1518,7 +1518,7 @@ def get_int_from_env(env_keys, default):
             # in_len.txt maybe shorter than we need,
             # use much longer context to make sure input length
             test_length = min(in_len*2, 8192)
-            while test_length not in [32, 256, 1024, 2048, 8192]:
+            while test_length not in [32, 256, 1024, 2048, 8192] and test_length < 8192:
```

Comment:
shall we also add this to other test_api ?

Reply:
merge this PR first, will add in other PR.